generated/nidcpower/nidcpower/session.py

```diff
@@ -4760,27 +4782,33 @@ def __init__(self, resource_name, channels=None, reset=False, options={}):
                 | driver_setup            | {}      |
                 +-------------------------+---------+
 
+            independent_channels (bool): Specifies whether to initialize the session with
+                independent channels. Set this argument to False on legacy applications or if you
+                are unable to upgrade your NI-DCPower driver runtime to 20.6 or higher.
+
 
         Returns:
             session (nidcpower.Session): A session object representing the device.
 
         '''
         super(Session, self).__init__(repeated_capability_list=[], vi=None, library=None, encoding=None, freeze_it=False)
+        resource_name = _converters.convert_repeated_capabilities_without_prefix(resource_name)
         channels = _converters.convert_repeated_capabilities_without_prefix(channels)
         options = _converters.convert_init_with_options_dictionary(options)
         self._library = _library_singleton.get()
         self._encoding = 'windows-1251'
 
         # Call specified init function
-        self._vi = 0  # This must be set before calling _initialize_with_channels().
-        self._vi = self._initialize_with_channels(resource_name, channels, reset, options)
+        self._vi = 0  # This must be set before calling _fancy_initialize().
```

Comment:
I too noticed this. Would be a quick line to change if you wanted it done, but looks like it has [been in the template](https://github.com/ni/nimi-python/blob/c631e3d8d034ac8c2ec8224363b78d5fd67b82af/build/templates/session.py.mako#L272) for a while.

Reply:
oh I remember why! it's because a pointer of self._vi is passed to _library when making the function call, and you don't want that to be garbage.

But doesn't Python initialize it to 0 anyway? 🤔

Reply:
This is out of scope and if we want to play with changing it it should be a separate PR anyway. Resolving.